cartridge/test-helpers/cluster.lua

```diff
@@ -91,6 +110,53 @@ function Cluster:initialize()
     end
 end
 
+function Cluster:configure_stateboard()
+    local stateboard_workdir = fio.pathjoin(self.datadir, 'stateboard')
+
+    if self.stateboard_entrypoint == nil then
+        local project_root = fio.dirname(fio.dirname(debug.sourcedir()))
```

Comment:
You assume that `debug.sourcedir` points to `cartridge/test-helpers/cluster.lua`, but after rocks install it'll be `.rocks/share/tarantool/cartridge/test-helpers/cluster.lua`